.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
```

Comment:
Let dependabot do some of the legwork for us when there are updates to dependencies.